addons/project/views/project_views.xml

```diff
@@ -61,7 +61,7 @@
             <field name="mode">primary</field>
             <field name="arch" type="xml">
                 <field name="stage_id" position="after">
-                    <field name="display_project_id" string="Project"/>
+                    <field name="display_project_id" string="Project" invisible="context.get('default_project_id')"/>
```

Comment:
Which point do you want to achieve by doing that? 🤔 

I am wondering if the field should not move in another view instead of using the invisible attribute with the `default_project_id` context key. 🤔

Reply:
I added this to hide project field from views open from specific project like we did with other views :innocent: